meta/runtime.yml

```diff
@@ -25,6 +25,10 @@ plugin_routing:
       redirect: cisco.iosxr.iosxr
     bgp_address_family:
       redirect: cisco.iosxr.iosxr
+    iosxr_bgp_neighbor_address_family:
```

Comment:
Once the _bgp_global, _af and _neighbor_af modules are merged, the old iosxr_bgp needs to be deprecated.

Reply:
Ack,made changes